### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by MTahir001**
Pls review the CSS project responsiveResume
& let me know if any changes are required, to make it perfect.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+  font-weight: 400;
+}
+.skills .box .percent {
+  position: relative;
+  width: 100%;
+  height: 10px;
+
+  background-color: #f0f0f0;
+}
+.skills .box .percent div {
```
Comment (OSAMA478) [inline style.css:206]: avoid using lengthy selectors

**Review by OSAMA478 — APPROVED** [review]
avoid using lengthy selectors....

**Review by ManSOorcode — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (MTahir001) [reply style.css:206]: for understanding purposes only I write like this

**Comment by syedabdullahali** [comment]
image

Try to center your text   or u can add contact info along with your Education  section

**Review by murtazshaikh — APPROVED** [review]
LGTM!

**Comment by MTahir001** [comment]
> image
> 
> Try to center your text or u can add contact info along with your Education section

done!!!

**Pull request merged**